lib/mail/encodings/8bit.rb

```diff
@@ -25,6 +25,11 @@ def self.cost(str)
         1.0
       end
 
+      # Per RFC 2821 4.5.3.1, SMTP lines may not be longer than 1000 octets including the <CRLF>.
+      def self.should_transport?(str)
```

Comment:
probably better named as `compatible_transport?` to indicate the purpose more clearly